manual-tests.md

```diff
@@ -50,6 +50,17 @@
 3. Run `snapcraft cleanbuild --remote <remote>` where `<remote>` is
    the name you gave the remote on step 1.
 
+
+# Test containerized building
+
+1. Setup a remote as described on
```

Comment:
The remote step here should now be gone and replaced with something about having a proper lxd setup